packages/devtools_shared/lib/src/deeplink/app_link_settings.dart

```diff
@@ -27,14 +33,18 @@ class AppLinkSettings {
   }
 
   /// Used when the the server can't retrieve app link settings.
-  static const empty = AppLinkSettings._('', <AndroidDeeplink>[]);
+  static const empty = AppLinkSettings._('', false, <AndroidDeeplink>[]);
 
   static const _kApplicationIdKey = 'applicationId';
+  static const _kDeeplinkingFlagEnabledKey = 'deeplinkingFlagEnabled';
   static const _kDeeplinksKey = 'deeplinks';
 
   /// The application id of the Android build of this Flutter project.
   final String applicationId;
 
+  /// The flag to enable deep linking.
```

Comment:
can you add more info to this dart doc? where is this flag from? is it set by the user for their project, or is this referring to the Deep link tool in DevTools?